src/main/java/graphql/ExecutionInput.java

```diff
@@ -187,6 +206,7 @@ public static Builder newExecutionInput(String query) {
         private String query;
         private String operationName;
         private Object context = GraphQLContext.newContext().build();
+        private GraphQLContext graphQLContext = GraphQLContext.newContext().build();
```

Comment:
Given that `context` is also an instance of `GraphQLContext` by default, perhaps the new `graphQLContext` can be initialized to the same instance, so that the new `graphQLContext` methods simply exposes the existing default `context` as `GraphQLContext`. 

This can help with migration for cases where the existing `context` is not replaced but rather downcast since code can be switched to simply access it through the new `graphQLContext` methods. And it shouldn't change anything for cases where the existing `context` is replaced.

Reply:
Great idea - thanks

Reply:
done